tests/web/Earthfile

```diff
@@ -0,0 +1,21 @@
+VERSION 0.6
+
+FROM ../..+earthly-integration-test-base --DOCKERHUB_AUTH=false
```

Comment:
can you follow the same pattern we use elsewhere:

https://github.com/earthly/earthly/blob/main/tests/config/Earthfile#L3-L13

i.e. passing along the dockerhub mirror credentials.

without this we will run into rate limiting issues.

Reply:
done